src/generator/AutoRest.Go/TemplateModels/ModelsTemplateModel.cs

```diff
@@ -15,6 +16,7 @@ public class ModelsTemplateModel : ServiceClient
         public List<ModelTemplateModel> ModelTemplateModels { get; set; }
         public string PackageName { get; set; }
         public Dictionary<IType, string> PagedTypes { get; set; }
+        public List<IType> NextMethodUndefined { get; set; }
```

Comment:
I added this for paged methods that its next method isn't defined in the swagger. How the generator handles this right now is adding a preparer in the models file, specific for the paged model. I am using this list to keep track of which models would need the preparer, so it is not generated when the next method is already defined. When the next method is defined in the swagger, the generator takes it as any other method and generates its preparer, sender and receiver next to the method and the other group methods.